app/lib/court_reminders_importer.rb

```diff
@@ -0,0 +1,52 @@
+module CourtRemindersImporter
+  def self.generate_reminders(court_dates, court_locations)
+    time_zone_offset = '-0600' # Make sure this is by instance eventually
+
+    Message.transaction do
+      Message.scheduled.auto_court_reminders.destroy_all
```

Comment:
How are we going to simulate this action on our first run? (destroying existing court reminders that weren't created with this process.)